whispersapi/serializers.py

```diff
@@ -4004,6 +3995,33 @@ def create(self, validated_data):
 
         return user
 
+    @staticmethod
+    def send_email_verification_message(user):
+        """Send email to user with link to verify their email address."""
+        # Create email verification link
+        token = email_verification_token.make_token(user)
+        verification_link = (settings.APP_WHISPERS_URL +
+                             "?" +
+                             urlencode({'user-id': user.id, 'email-token': token}))
+
+        # create a 'User Email Verification' notification
+        # source: User that requests a public account
+        source = user.username
+        # recipients: user
+        recipients = [user.id]
+        # email forwarding: Automatic, to user's email
+        email_to = [user.email]
+        # TODO: add protection here for when the msg_tmp is not found (see scheduled_tasks.py for examples)
+        msg_tmp = NotificationMessageTemplate.objects.filter(name='User Email Verification').first()
```

Comment:
Could you put in a code block here to check if the msg_tmp exists? I've been doing that for all other message template look ups, so it would be good to have it here too for consistency.

Reply:
Yes, will do